pom.xml

```diff
@@ -31,19 +31,19 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.product</groupId>
       <artifactId>partysvc-api</artifactId>
-      <version>10.50.2</version>
+      <version>10.50.3</version>
```

Comment:
Need to change this to 10.50.4

Reply:
the release version with the update to the json ignore is 10.50.3, travis has bumped the snapshot version up to to 10.50.4 for development.